frontend/app/components/CardBack.tsx

```diff
@@ -36,48 +33,130 @@ export const CardBack = ({
         {
           width: CARD_WIDTH,
           height: CARD_HEIGHT,
-          backgroundColor: '#151312',
+          backgroundColor: '#1c1a2d',
         },
         backAnimatedStyle,
       ]}
     >
-      <Text style={styles.content}>{item.content}</Text>
+      <ScrollView
+        contentContainerStyle={styles.scrollContent}
+        showsVerticalScrollIndicator={false}
+        keyboardShouldPersistTaps="handled"
+      >
+        <Text style={styles.title}>{item.content}</Text>
+
+        <Animated.View style={styles.divider} />
+        <Section title="Meaning" content={item.meaning} />
+        <Section title="Explanation" content={item.explanation} />
+        <Section title="Examples" content={item.examples} />
+      </ScrollView>
       <TouchableOpacity
         onPress={handleFavoritePress}
-        style={[
-          styles.iconButton,
-          {
-            bottom: CARD_HEIGHT * 0.05,
-            right: CARD_WIDTH * 0.05,
-          },
-        ]}
+        style={styles.favoriteButton}
+        activeOpacity={0.7}
       >
         <Ionicons
           name={item.isFavorite ? 'star' : 'star-outline'}
-          size={28}
-          color={item.isFavorite ? '#FFD700' : '#FFFFFF'}
+          size={26}
+          color={item.isFavorite ? '#FFD700' : 'rgba(255, 255, 255, 0.8)'}
         />
       </TouchableOpacity>
     </Animated.View>
   );
 };
 
+const Section = ({
+  title,
+  content,
+}: {
+  title: string;
+  content: string | string[];
+}) => {
+  const renderContent = () => {
+    if (title === 'Examples') {
+      let examplesList: string[] = [];
+
+      if (typeof content === 'string') {
+        examplesList = content.split('\n').filter(Boolean);
+      } else if (Array.isArray(content)) {
+        examplesList = content.filter(Boolean);
+      }
```

Comment:
This is an anti-pattern, we don't want to change the component dynamically, we want to use the correct component for each content type directly in the code (it will get fixed when we do separate components).